notebooks/infer_predictions.ipynb

```diff
@@ -0,0 +1,180 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Run inference on pre-trained model:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "import torch"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from grace.io.image_dataset import ImageGraphDataset\n",
+    "from grace.models.feature_extractor import FeatureExtractor\n",
+    "from grace.evaluation.visualisation import plot_simple_graph\n",
```

Comment:
I get an error trying to run this cell:
```ModuleNotFoundError: No module named 'grace.evaluation'``` 

I think there is a missing __init__.py in the evaluation directory that would make this work.

Reply:
Adding the `__init__.py` in the next commit 🚀